tests/unit/amo/components/Errors/TestErrorComponent.js

```diff
@@ -0,0 +1,61 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+
+describe(__filename, () => {
+  function render(props = {}) {
+    return shallow(<ErrorComponent {...props} />);
+  }
+
+  it('renders an ErrorComponent', () => {
+    const root = render();
+
+    expect(root.find(NestedStatus)).toHaveProp('code', 400);
+    expect(root.find(Card)).toHaveClassName('Errors');
+  });
+
+  it('renders header when header is available', () => {
+    const root = render({ header: 'foo' });
+
+    expect(root.find(Card)).toHaveProp('header', 'foo');
+  });
+
+  it('renders code when code is supplied', () => {
+    const root = render({ code: 500 });
+
+    expect(root.find(NestedStatus)).toHaveProp('code', 500);
+  });
+
+  it('renders children', () => {
+    const root = render({ children: 'hello' });
```

Comment:
As discussed above, using a `const` for `children` and using it in both the `render` and the `expect` is better practice.